docker/loom-delta-test/requirements.txt

```diff
@@ -0,0 +1,8 @@
+Click==7.0
```

Comment:
I don't see `Click` is used anywhere in the PR, it's all `argparse` based scripts 🤔

Reply:
yes for some reason pip freeze showed it and i was reluctant to remove, maybe a dependency requires it?

Reply:
Not sure, it's not harmful so let's keep it :)